doc/api/esm.md

```diff
@@ -62,9 +62,9 @@ or when referenced by `import` statements within ES module code:
 
 ## <code>package.json</code> <code>"type"</code> field
 
-Files ending with `.js` or `.mjs`, or lacking any extension,
-will be loaded as ES modules when the nearest parent `package.json` file
-contains a top-level field `"type"` with a value of `"module"`.
+Files ending with `.js` or lacking any extension will be loaded as ES modules
```

Comment:
I'd second @geoffreybooth here, the list leaving out one of the types means it should be defined as a delta from the default behavior. Perhaps changing that would be fine.

@tamias , 

> Files ending in .js, or extensionless files, when the nearest parent
package.json file contains a top-level field "type" with a value of
"module".

Does show duplication elsewhere regarding .js files in the section being changed. I'm unclear how the line about .mjs files is pertinent to be removed due to

> Files ending in .mjs

Which is colocated with the other duplicated bit of documentation, but .js is not.

I think unless there is reason to remove both duplicated point I don't find it a good idea to remove either.

Reply:
@bmeck Could you clarify what duplication you're referring to regarding .js files?

Reply:
https://github.com/nodejs/node/blob/e66f770afa77029eb0a6b465a34a04738d9d78f3/doc/api/esm.md#enabling contains information on .js files in a similar manner to the .mjs documentation you pointed to as being a reason we don't need to include .mjs in the type:"module" explainer. It seems if the documentation in `enabling` replaces the need for the contents of the explanation of `type:"module"` it would apply to both .js and .mjs needing to be remove not solely .mjs needing to be removed .

Reply:
The reason not to include .mjs in the `type: "module"` explainer is that `type: "module"` **doesn't affect the handling of .mjs files**.  The current documentation, however, suggests that the handling is affected.

It's not about removing duplication; it's about removing ambiguity.

Reply:
@tamias The docs state how things work within the scope of `"type":"module"`; not how things are altered. If you rephrased how things are altered it would certainly change the framing of the docs but would still mean that we should remove the points from `enabling` about `"type":"module"` as well if the problem is being ambiguous. I do not see any ambiguity here, but would be open to rephrasing if something is found. What is ambiguous exactly?